backend/LexBoxApi/Services/ProjectService.cs

```diff
@@ -123,7 +124,15 @@ public async Task UpdateProjectMetadata(string projectCode)
         var count = await hgService.GetLexEntryCount(projectCode);
         if (project.FlexProjectMetadata is null)
         {
-            project.FlexProjectMetadata = new FlexProjectMetadata { LexEntryCount = count };
+            try
+            {
+                project.FlexProjectMetadata = new FlexProjectMetadata { LexEntryCount = count };
```

Comment:
I don't think this is where the exception is thrown. Can you reproduce the issue and force it to error to validate that this fixes the problem? Just replace `if (project.FlexProjectMetadata is null)` above with `if (true)` to test out the condition

Reply:
Done in commit ce10f64.